.github/labeller.yml

```diff
@@ -0,0 +1,10 @@
+# Labeler configuration file for GitHub Actions
```

Comment:
the file name has an extra `L`

Reply:
I guess this is inconsistent though with the docs. I'll change it.

Reply:
EDIT. I'm as dumb as a stump today. Of course it has to be. Done.